### DISCUSSION TITLE
Is it possible to implement the Next.js PR stats workflow in a Next.js / React codebase

### DISCUSSION THREAD

**Discussion by luke-h1** [vercel/next.js#28075] [Help]
Hi, I want to implement some performance testing (like the PR stats workflow does in the next.js repo) in order to measure things such as build time, page load times, etc. I checked out https://github.com/vercel/next-stats-action however I noticed that it is archived. I'm just wondering if it's possible to still use the stats app to measure the performance of a next.js / React codebase via GitHub workflows?

**Comment by stovmascript** [comment]
The action now lives here: https://github.com/vercel/next.js/tree/canary/.github/actions/next-stats-action.

I'm also curious how to go about implementing this in our project.

**Reply by stovmascript** [reply]
I'm not sure how to reference it in the `uses` field of my workflow. Also, it seems that it's missing an `action.yml` file.
